dist/trackers.json

```diff
@@ -19877,7 +19877,7 @@
             "url": "http://www.yume.com/",
             "companyId": "yume"
         },
-        "yume,_inc.": {
+        "yume_inc": {
```

Comment:
Actually, it would be better to fix the script that generates the CSV, not the data (since it came from whotracksme).

Reply:
Add escaping at build for such titles or exclude characters that are worth escaping?

Reply:
Better add escaping